features/page_objects/deregistration_page.rb

```diff
@@ -3,21 +3,35 @@ class DeregistrationPage < SitePrism::Page
   element(:deregister_status_dropdown, "#admin_deregister_enrollment_exemption_form_status")
   element(:revoked_option, "option[value='revoked']")
   element(:ceased_option, "option[value='ceased']")
-  element(:deregistration_comment, "#admin_deregister_enrollment_form_comment")
+  element(:reg_dereg_comment, "#admin_deregister_enrollment_form_comment")
+  element(:exemption_dereg_comment, "#admin_deregister_enrollment_exemption_form_deregistered_comment")
 
   element(:deregister_button, "input[name='commit']")
 
   section(:nav_bar, AdminNavBarSection, ".add-bottom-margin .container")
 
-  def submit(args = {})
+  def deregister_registration(args = {})
```

Comment:
It looks like we're wrapping both the exemption deregistration page and the registration deregistration page into one page object. These should be split into distinct page objects.

Also I'd stick with the pattern of having a `submit()` method into which we pass in the values we wish to enter/select on a page from the step, and then let that method handle what needs to be done, rather than naming it specific to the page/feature.